### ISSUE TITLE
Parser: header line without info causes valid yarn files to break

### ISSUE THREAD

**Issue by Schroedingers-Cat** [YarnSpinnerTool/YarnSpinner#189]
Importing a yarn file in Unity with a header line without content causes a bug in the parser to break at syntactically valid instructions. Examples are:
```
title: Start
tags: 
colorID: 0
position: 0,0
parent: 
---
Test
===
```
or
```
title: Start
tags: 
colorID: 0
position: 0,0
parent: 
---
A: Test
// You can make comments with '//' and the player won't see it.
===
```
Removing the `parent: ` line or adding content to it like `parent: 123456`

**Comment by desplesda** [comment]
A complete fix for this will involve updating how the parser handles headers, which I'd like to tweak. I've started on this by adding some tests for how headers are parsed, in 8736b3b4e.

**Comment by desplesda** [comment]
Fix will be a part of v1.1.
